File: Java-SpringBoot/BE-02-DB/java-21-springboot-4.0/src/main/java/com/flyrank/taskservice/dto/ErrorResponse.java
package com.flyrank.taskservice.dto;

import io.swagger.v3.oas.annotations.media.Schema;

@Schema(description = "Standard error response payload")
public record ErrorResponse(
        @Schema(description = "Readable error message detail", example = "Task with id '999' not found")
        String detail
) {}

File: Java-SpringBoot/BE-02-DB/kotlin-2.3-springboot-3.5/src/main/kotlin/com/flyrank/taskservice/dto/ErrorResponse.kt
package com.flyrank.taskservice.dto

import io.swagger.v3.oas.annotations.media.Schema

@Schema(description = "Standard error response payload")
data class ErrorResponse(
    @field:Schema(description = "Readable error message detail", example = "Task with id '999' not found")
    val detail: String
)
